src/web/components/Subtitle/index.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default function Subtitle({ children, level }) {
+  const raisedLevel = level + 1;
+  const classes = `subtitle is-${raisedLevel}`;
+  const Heading = `h${raisedLevel}`;
+  return <Heading className={classes}>{children}</Heading>;
```

Comment:
No error on eslint at that line ?

Reply:
Nop for me.

Reply:
Yes, it works smile .
Initially I used a switch case for the tests, but I couldn't get 100% coverage without doing a test per case. So I did a little research on Stackoverflow and I found this solution.

I'm gonna refactor level.